packages/client/src/utils/constants.ts

```diff
@@ -94,10 +94,10 @@ export const MONTHS_IN_YEAR = 12
 
 /* change to import.meta.env.mode when migrating to vitest */
 export const LOADER_MIN_DISPLAY_TIME =
-  process.env.NODE_ENV !== 'test' ? 3 * 1000 : 0 // 3 seconds except test environment
+  import.meta.env.NODE_ENV !== 'test' ? 3 * 1000 : 0 // 3 seconds except test environment
```

Comment:
@tahmidrahman-dsi We probably want to use `env.MODE` here instead

Reply:
Thanks @Zangetsu101 , done in https://github.com/opencrvs/opencrvs-core/pull/4047/commits/1dce985488c2886aa2c1b08400e82cc30c2eceea